frida-gum-sys/build.rs

```diff
@@ -158,6 +165,27 @@ fn main() {
             .compile("stalker_observer");
     }
 
+    #[cfg(feature = "stalker-params")]
+    {
+        let mut builder = cc::Build::new();
+
+        #[cfg(feature = "auto-download")]
+        #[allow(unused_mut)]
+        let mut builder = builder.include(include_dir);
+
+        #[cfg(not(feature = "auto-download"))]
+        let builder = if std::env::var("DOCS_RS").is_ok() {
+            builder.include("include")
+        } else {
+            &mut builder
+        };
+
+        builder
+            .file("stalker_params.c")
+            .opt_level(3)
```

Comment:
Question here: doesn't bindgen do this automatically when we are in release mode?

Reply:
You mean the `opt_level(3)`? I'm not sure tbh. I copied the template used by the other files above in the `build.rs`. I guess if one is changed, then they should all be? Perhaps extracting a method could reduce the code duplication too?